### ISSUE TITLE
Consolidate download links

### ISSUE THREAD

**Issue by confit** [super3/peercoin_website_old#298]
The Windows and Linux wallets are hosted on SourceForge and the Mac wallet is hosted on mega.co.nz.  Consolidate downloads into one location.

It isn't clear who maintains or has access to files on a mega.co.nz account.  Furthermore, it isn't clear why the Mac wallet maintainer(s) cannot access "official" Peercoin infrastructure (Peercoin's SourceForge account or peercoin.net).

**Comment by iheartcryptocoin** [comment]
Agree.  We also need to consolidate a lot more than this.  Rather than do the consolidations piecemeal as we have been, how about we make an "consolidation" issue and all comment with the changes we see until we are happy with the "master list" of all the consolidations that need to be done. Then we each can choose which large chunk we want to own?

**Comment by super3** [comment]
See #288
